crates/cargo-lambda-watch/src/lib.rs

```diff
@@ -35,6 +39,10 @@ pub struct Watch {
     #[clap(long)]
     no_reload: bool,
 
+    /// Address where users send invoke requests
+    #[clap(short = 'a', long, default_value = "::")]
```

Comment:
To figure out if `"::"` or `"[::]"` was right, I wrote the following code. The code below printed `[::]:9000`, which is what we want. If I switched it to `"[::]"` I received a runtime error.

```rust
    let ip = IpAddr::from_str("::").unwrap();
    let addr = SocketAddr::from((ip, 9000_u16));

    println!("{}", addr);
```